test/test_CE6SSM_gmm2.cpp

```diff
@@ -127,62 +105,29 @@ Block EXTPAR
 
    // extract the input parameters
    softsusy::QedQcd qedqcd;
-   Physical_input physical_input;
    CE6SSM_input_parameters input;
    Spectrum_generator_settings settings;
 
    try {
       slha_io.fill(qedqcd);
       slha_io.fill(input);
-      slha_io.fill(physical_input);
       slha_io.fill(settings);
    } catch (const Error& error) {
       BOOST_TEST_MESSAGE(error.what());
       BOOST_TEST(false);
    }
 
-   //std::cout << input.LambdaInput << std::endl;
-
-   //settings.set(Spectrum_generator_settings::calculate_sm_masses, 0);
-   //settings.set(Spectrum_generator_settings::calculate_bsm_masses, 0);
-
    CE6SSM_spectrum_generator<Semi_analytic> spectrum_generator;
    spectrum_generator.set_settings(settings);
    spectrum_generator.set_parameter_output_scale(slha_io.get_parameter_output_scale());
 
    spectrum_generator.run(qedqcd, input);
 
-   CE6SSM_scales scales;
-   scales.HighScale = spectrum_generator.get_high_scale();
-   scales.SUSYScale = spectrum_generator.get_susy_scale();
-   scales.LowScale  = spectrum_generator.get_low_scale();
-   scales.pole_mass_scale = spectrum_generator.get_pole_mass_scale();
-
-
-   /*CE6SSM_mass_eigenstates m;
-
-   //setup_CE6SSM_const(m, input);
-
-   std::cout << "vd" << m.get_vd() << std::endl;
-   std::cout << "vu" << m.get_vu() << std::endl;
-   std::cout << "Cha " << m.get_mass_matrix_Cha() << std::endl;
-   std::cout << "Chi " << m.get_mass_matrix_Chi() << std::endl;
-   std::cout << "Se " << m.get_mass_matrix_Se() << std::endl;
-   std::cout << "Sv " << m.get_mass_matrix_Sv() << std::endl;*/
-   std::cout << "qedqcd " << qedqcd << std::endl;
-
    auto models = spectrum_generator.get_models_slha();
-
-   auto model_at_scale = std::get<0>(models);
-   model_at_scale.run_to(scales.pole_mass_scale);
-
-   CE6SSM_effective_couplings effective_couplings(model_at_scale, qedqcd, physical_input);
-   effective_couplings.calculate_effective_couplings();   
-
-   auto amu = CE6SSM_a_muon::calculate_a_muon(model_at_scale, qedqcd);
+   auto amu = CE6SSM_a_muon::calculate_a_muon(std::get<0>(models), qedqcd);
 
    constexpr double reference_value = 1.82135849E-11;
```

Comment:
Would it be possible to write this reference value as a sum of two values:
* The value w/o your new diagrams plus
* the contribution from your diagrams.

Like so:
~~~.cpp
constexpr double reference_value =
   + 1.7E-11  // contribution from FFS + SSF, see [1710.03760]
   + 0.1E-11; // contribution from [your diagram types here]
~~~

Reply:
@Expander This has now been implemented.  Currently using value from FlexibleSUSY, working on SPheno calculation.

Reply:
Many thanks for this!